apps/web/components/PoolGovernance.tsx

```diff
@@ -72,7 +74,7 @@ export const PoolGovernance = ({
                   <div className="flex w-full items-center gap-6">
                     <h5 className="">Status:</h5>
                     <div>
-                      <StatusBadge status={isMemberActivated ? 1 : 0} />
+                      <Badge status={isMemberActivated ? 1 : 0} />
```

Comment:
We need to think of a better way to handle pool types and proposal, Ill think about something to push in another PR once all the current one will be merged, I have some ideas with type enforced and no use of numbers never

Reply:
https://github.com/1Hive/gardens-v2/issues/263